src/components/form/signup.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Button from './button'
+import Input from './input'
+import Row from '../containers/row'
+import Facebook from 'images/svg/facebook.svg'
+import Google from 'images/svg/google.svg'
+
+const Title = styled.h2`
```

Comment:
Is this title style used at other pages? if so let's use a shared `<Header />`

Reply:
handed to @arash-binary